### PR TITLE
Added note about when to assign the license to perform-G-Suite-migration.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/2478.

### CODE REVIEW

**Comment by dariomws** [comment]
I've verified that once the MailUser is converted to Mailbox, it appears as required license reconciliation with the following cmdlet: 
Get-MsolUser -LicenseReconciliationNeededOnly

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md [hunk 1]
```diff
@@ -264,7 +264,10 @@ We recommend that the primary address (sometimes referred to as the "User ID") f
 
     ![Schedule batch migration](../media/schedule-batch1-migration.png)
     
-    The batch status changes from **Syncing** to **Synced**, you can complete the batch. 
+    Once the batch status changes from **Syncing** to **Synced**, you can complete the batch.
+    
+    > [!NOTE]
+    > When the batch starts, all the users to be migrated will be converted from MailUsers to Mailboxes. The Microsoft 365 or Office 365 Exchange license must be assigned only after this moment. You have 30 days to assign the license.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md:270]: @dariomws - may I suggest that we change "... must be assigned only after this moment." to "... must be assigned only after this conversion."?

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**